plugins/teranaut/index.js

```diff
@@ -101,13 +106,19 @@ function ensureAuthenticated(req, res, next) {
     // We allow creating new accounts without authentication.
     const { token } = req.query;
     if (teranaut.auth.open_signup) {
-        if (req.url === '/accounts' && req.method === 'POST') return next();
+        if (req.url === '/accounts' && req.method === 'POST') {
+            next();
+            return;
```

Comment:
What's the reason for all these changes around return values?

Reply:
To fix linting issues about consistent returns